react-app/src/api/useGetSystemNotifs.ts

```diff
@@ -0,0 +1,49 @@
+import { API } from "aws-amplify";
+import { useState, useEffect } from "react";
+import { useGetUser } from "@/api";
+import { useQuery } from "@tanstack/react-query";
+import { BannerNotification, ReactQueryApiError } from "shared-types";
+
+export const getSystemNotifs = async (): Promise<BannerNotification[]> => {
+  return await API.get("os", "/systemNotifs", {});
+};
+
+export const useGetSystemNotifs = () => {
+  const userQuery = useGetUser();
+  const [dismissed, setDismissed] = useState<string[]>([]);
+
+  useEffect(() => {
+    const dismissedNotifs = localStorage.getItem(`notifs.${userQuery?.data?.user?.username}`);
+    const parsed: string[] = JSON.parse(dismissedNotifs) ?? [];
```

Comment:
This is the only issue I see. `JSON.parse` throws an error if something's not right, so perhaps it's a good idea to handle corrupt JSON. But if you don't suspect anything happening, I guess you can leave it